tests/examples/example_specs/components/component.yaml

```diff
@@ -3,19 +3,15 @@ description: This is an example component
 image: example_component:latest
 
 consumes:
-  images:
-    fields:
-      data:
-        type: binary
+  images_data:
+    type: binary
 
 produces:
-  embeddings:
-    fields:
-      data:
-        type: array
-        items:
-          type: float32
-    additionalFields: false
+  images_data:
+    type: array
+    items:
+      type: float32
+additionalFields: false
```

Comment:
This was removed for now, right?

Reply:
Would leave it here for now. The file is moved to a different folder anyway in #655.